bin/postinstall.sh

```diff
@@ -3,10 +3,12 @@ id -u iip &> /dev/null || useradd iip
 
 # create directory for logging
 mkdir -p /var/log/iip
-
-# give permission to iip user
 chown -R iip:iip /var/log/iip
 
+# create directory for work dir
+mkdir -p /data
```

Comment:
This directory should be handled by the admins, since it will be a separate filesystem.